pom.xml

```diff
@@ -200,7 +200,7 @@
     <picocli.version>4.6.3</picocli.version>
     <prometheus.version>0.9.0</prometheus.version>
     <protobuf.version>3.20.1</protobuf.version>
-    <quarkus.version>2.8.2.Final</quarkus.version>
+    <quarkus.version>2.9.0.Final</quarkus.version>
```

Comment:
just wondering, should we maybe rather ignore the scala stuff from the quarkus-bom?

Reply:
If only that would work :(